app/models/order.rb

```diff
@@ -28,4 +21,18 @@ def self.cancelled
   def self.ordered
     all.select { |order| order.status == 'ordered'}
   end
+
+  private
+
+  def calculate_new_order_total(loan)
```

Comment:
Unused method argument - `loan`. If it's necessary, use `_` or `_loan` as an argument name to indicate that it won't be used. You can also write as `calculate_new_order_total(*)` if you want the method to accept any arguments but don't care about them.